front/app/components/admin/AddCollaboratorsModal/index.tsx

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+
+// Components
+import { Box, Button, Text } from '@citizenlab/cl2-component-library';
+import Modal from 'components/UI/Modal';
+import SeatInfo from 'components/SeatInfo';
+
+// Translation
+import { FormattedMessage, useIntl } from 'utils/cl-intl';
+import messages from './messages';
+
+// hooks
+import useAppConfiguration from 'api/app_configuration/useAppConfiguration';
+import useSeats from 'api/seats/useSeats';
+
+// Utils
+import { isNil } from 'utils/helperUtils';
+
+interface Props {
+  showModal: boolean;
+  closeModal: () => void;
+  addModerators: () => void;
+  // Total number of seats to add, including admin seats
+  noOfSeatsToAdd: number;
+  // These don't include admin seats
+  noOfSeatsToBuy?: number;
+}
+
+const AddCollaboratorsModal = ({
+  showModal,
+  closeModal,
+  addModerators,
+  noOfSeatsToAdd,
+  noOfSeatsToBuy = noOfSeatsToAdd,
```

Comment:
I find this default value confusing. Could you elaborate with a comment?

At first sight, it also seems wrong. If we have spots available, noOfSeatsToBuy should be 0, regardless of noOfSeatsToAdd.

Reply:
Similar, I think the naming is confusing but still related to admins so I'll make the change after. But ideally this means that if no seats to buy are passed, then the seats to buy are the same as the seats you want to add